run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+WORKING_DIRECTORY=$(pwd)
+echo "Starting up in $WORKING_DIRECTORY"
+
+# try watching interval
+if [ "$1" == "" ]; then
+    INTERVAL="15s"
+else
+    INTERVAL=$1
+fi
+echo "Setting update interval to $INTERVAL"
+
+# start Flask API
+echo "Starting Flask..."
+flask run -p 80 -h 0.0.0.0 & sleep 1s
+
+# start Celery worker
+echo "Starting Celery worker..."
+celery -A spectre_analyses worker --loglevel=info -n divik-worker@%h &
+
+# watch them running
+while sleep $INTERVAL; do
+
+    ps aux | grep flask | grep -q -v grep
+    FLASK_DOWN=$?
+    if [ $FLASK_DOWN -ne 0 ]; then
+        echo "Flask died."
+        exit -1
+    fi
```

Comment:
Using `pgrep`, the entire Flask check can be simplified to:
```sh
if ! pgrep flask > /dev/null; then
  # failure detected
fi
```
Same should be applied for Celery.

Reply:
Common POSIX error codes are positive, so use `exit 1` for Flask.

Reply:
Accepted.